app/controllers/join.js

```diff
@@ -12,6 +12,7 @@ export default class JoinController extends Controller {
   @tracked chaincode = 'Generate chaincode';
   @tracked isChaincodeClicked = false;
   @tracked isLoading = false;
+  ANKUSH_TWITTER = ANKUSH_TWITTER;
```

Comment:
This re-declaration of same variable is not needed we can skip and directly use the imported one

Reply:
resolving this since it's been used in hbs file so this is needed